runtime/android/core/strings/xwalk_app_strings.grd

```diff
@@ -29,6 +29,12 @@
       <message desc="Startup Not Found Message [CHAR-LIMIT=100]" name="IDS_STARTUP_NOT_FOUND_MESSAGE">
         Please get Crosswalk Runtime Library
       </message>
+      <message desc="Startup Verify Error Title [CHAR-LIMIT=100]" name="IDS_STARTUP_VERIFY_ERROR_TITLE">
+        Crosswalk Signature Verification Failed
+      </message>
+      <message desc="Startup Verify Error Message [CHAR-LIMIT=100]" name="IDS_STARTUP_VERIFY_ERROR_MESSAGE">
+        Please uninstall current Crosswalk Runtime Library
```

Comment:
That doesn't sound like an appropriate error message. Aren't you supposed to inform the user that the integrity of the runtime package could not be verified and thus you're aborting, and then perhaps provide some instructions on how to fix the situation?